test/index.bs

```diff
@@ -203,27 +208,38 @@ When invoked, the {{USBTest/reset()}} method MUST return a new {{Promise}}
 |promise| and run the following steps <a>in parallel</a>:
 
 1.  Let |test| be the {{USBTest}} instance on which this method was invoked.
-1.  Set |test|.{{USBTest/chosenDevice}} to <code>null</code>.
 1.  For each {{FakeUSBDevice}} |fakeDevice| previously returned by
     {{USBTest/addFakeDevice()}}, invoke
     |fakeDevice|.{{FakeUSBDevice/disconnect()}}.
 1.  <a>Resolve</a> |promise|.
 
+When invoked, the {{USBDeviceRequestEvent/respondWith(result)}} method MUST run
+the following steps <a>in parallel</a>:
+
+1.  Wait until |result| settles.
+1.  Let |device| be <code>undefined</code>.
+1.  If |result| resolved with |response| and |response| is an instance of
+    {{FakeUSBDevice}} let |device| be |response|.
+1.  Resume execution of the suspended steps defined for {{USB/requestDevice()}}.
```

Comment:
You can't resume execution from 'in parallel' steps. Instead, [queue a task](https://html.spec.whatwg.org/multipage/webappapis.html#queue-a-task) to resume execution.

Reply:
I've redefined this to be a lot more specific about holding onto the promise returned by requestDevice() instead of using a vague concept of "resuming execution." Please take a look.